projects/common/src/lib/symbology/raster-symbology-editor/raster-symbology-editor.component.html

```diff
@@ -1,33 +1,5 @@
-<geoengine-sidenav-header>Symbology Editor</geoengine-sidenav-header>
-
 <div class="outer" fxLayout="column">
     <div fxFlex class="container">
-        <geoengine-dialog-help>
-            <p>The Symbology Editor enables customization of the style for raster layers.</p>
-            <p>
-                The <i>Global Layer Properties</i> define the default visualization parameters. The layer <i>Opacity</i> is adjustable in a
-                range from 0 to 100 %. You can choose a <i>NoData</i> color for pixels with the nodata value. Use the picker tool to select
-                the desired RGB color and opacity. This also applies to the <i>Overflow</i> color, which indicates the pixels with values
-                without coloring rules.
-            </p>
-            <p>
-                The <i>Color Map</i> section provides an overview of the pixel values with a frequency plot, which also allows to adapt the
-                color with respect to the raster values. The plot refers to the field of view shown in the map. If
-                <i>Sync map and histogram </i> is turned on, the histogram updates if the map view changes. To specify the value range of
-                interest, you can set a minimum and maximum pixel value. You can choose a color ramp from a variety of color schemes (<i
-                    >Colormap name</i
-                >) and reverse it, if desired. Additionally, different functions for the step distribution can be selected (linear,
-                logarithmic, square root function, square function). Consider that the logarithmic function requires positive values (>0).
-                The number of Color steps is also kept flexible and can be set to a number between 2 and 16. Click
-                <i>Create color table</i> to apply your adjustments.
-            </p>
-            <p>
-                The <i> Color Table section </i> allows fine grained changes to colors. The gradient defines the interpolation between
-                values. You can dynamically add and remove color steps by clicking the minus and plus symbols or select distinct RGBA values
-                for a specific color step value.
-            </p>
-        </geoengine-dialog-help>
```

Comment:
Why is this gone? Where can users find help?

Reply:
the help is still in the core, because imho the geoengine-dialog-help makes more sense there. If you think we need the help not only in the core-based apps but also in the manager, we need to move the geoengine-dialog-help to common or do something else. idk